src/action.h

```diff
@@ -5,6 +5,7 @@
 #include <map>
 #include <string>
 #include <set>
+#include "enums.h"
```

Comment:
It seems it would be enough to forward declare struct tripoint (or was it a class?) without including the header here.